mne_bids_pipeline/steps/preprocessing/_08a_apply_ica.py

```diff
@@ -196,7 +196,12 @@ def apply_ica_raw(
     run: str,
     task: str | None,
     in_files: dict,
-) -> dict:
+) -> dict | None:
+    if cfg.spatial_filter_raw is None and not (cfg.task == "rest" or cfg.task_is_rest):
+        return
+    elif cfg.spatial_filter_raw is False:
+        return
```

Comment:
Rather than doing it at this level I would do it at the level of the `main` below. I'll push a commit